xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -242,7 +242,7 @@ viewers.Comments = Class.create({
    * Inline reply: Move the form under the replied comment and update the hidden "replyto" field.
    */
   addReplyListener : function() {
-    if (this.form) {
+    if ($("commentscontent")) {
```

Comment:
What's `commentscontent` refers to?
When looking on the else part of this if, it says:
>  // If, for some reason, the form is missing, hide the reply functionality from the user

I have the feeling that the whole if is now useless no? And the "for some reason" is a bit weird.

Reply:
I have this feeling but until I'm 200% I'll keep the else branch. The PR now proposes an alternative fix that is IMHO less prone to create unexpected regressions